client/coral-framework/components/Slot.js

```diff
@@ -32,27 +33,29 @@ class Slot extends React.Component {
   }
 
   getSlotProps(props = this.props) {
-    const {
-      fill: _a,
-      inline: _b,
-      className: _c,
-      reduxState: _d,
-      defaultComponent_: _e,
-      queryData: _f,
-      childFactory: _g,
-      component: _h,
-      ...rest
-    } = props;
-    return rest;
+    return omit(props, [
+      'fill',
+      'inline',
+      'className',
+      'reduxState',
+      'singleSlot',
+      'defaultComponent_',
+      'queryData',
+      'childFactory',
+      'component',
+    ]);
   }
```

Comment:
I see this hiding from the linter 😉

Reply:
Yeah I initially avoided `omit` apparently it utilises this triggers the inclusion of a chunk of lodash, but even without `omit` we already do, so that's fine.